src/components/LocalFileUpload/index.tsx

```diff
@@ -70,7 +70,12 @@ const LocalFileUpload: React.FC<FileUploadProps> = ({
                 }}
             >
                 {children || (
-                    <Button className="secondary-button">Select file</Button>
+                    <Button
+                        className="secondary-button"
+                        ant-click-animating-without-extra-node={false}
```

Comment:
when you select a file, this prop briefly flashes to true, and wipes out the border on the button, setting it to false maintains our styling